### PR TITLE
Update skill tree to current status

### PR DESCRIPTION
**Pull request by oli-obk**

### CODE REVIEW

File: src/skill_tree.md [hunk 1]
```diff
@@ -72,22 +78,46 @@ href = "https://github.com/rust-lang/rust/issues/53020"
...
+label = "unconst operations"
 href = "https://github.com/rust-lang/const-eval/issues/14"
+items = []
+requires = ["unsafe_rules"]
+
+[[group]]
+name = "transmute"
+label = "feature:const_fn_transmute"
+items = []
+requires = ["unconst_rules"]
```
Comment (RalfJung) [inline src/skill_tree.md:98]: Why does "transmute" require "unconst" when the very similar union/raw-ptr require "unsafe"?

Comment (oli-obk) [reply src/skill_tree.md:98]: hmm overlooked that one

Comment (oli-obk) [reply src/skill_tree.md]: force pushed the last commit

**Comment by RalfJung** [comment]
Thanks, looking good now. :)

**Pull request merged**